pkg/swap/swap.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
please use `sh` not `bash` - if a script can't be written in Posix shell it should be written in a real programming language...